packages/affiliates/apps/CreateDataSet.js

```diff
@@ -22,4 +22,5 @@ async function Run(config) {
 const config = Config();
 Run(config)
   .then(console.log)
-  .catch(console.error);
+  .catch(console.error)
+  .finally(() => process.exit());
```

Comment:
for some reason process does not exit. I think web3 or bigquery is holding a connection open but dont know how to close them. So this forces exit.